### PR TITLE
docs:policylibrary: more examples on privileges escalation and execution

### PR DESCRIPTION
**Pull request by tixxdz**
policylibrary: Privileges Escalation, unlinked and fileless binary execution

### CODE REVIEW

File: docs/content/en/docs/policy-library/observability/_index.md [hunk 1]
```diff
@@ -30,95 +33,170 @@ description: >
...
 
-### Policy
+Detecting the execution of `setuid` and `setgid` binaries is a common
+best-practice as attackers may abuse such binaries, or even create them
+during an exploit for subsequent execution.
 
-[library.yaml](https://raw.githubusercontent.com/cilium/tetragon/main/examples/policylibrary/library.yaml)
+### Requirement
 
+Run Tetragon with `enable-process-creds` setting set to enable visibility
```
Comment (sharlns) [inline docs/content/en/docs/policy-library/observability/_index.md:105]: This configuration level doesn't belong to the "Policy Library" section. Can you please point just a reference here to the documentation where you can enable credentials in processX events?

File: docs/content/en/docs/policy-library/observability/_index.md [hunk 2]
```diff
@@ -30,95 +33,170 @@ description: >
...
+man page; section `File capabilities`.
+
+Detecting the execution of `file capabilities` binaries is a common
+best-practice, since they cross privilege boundaries which make them a suitable
+target for attackers. Such binaries are also used by attackers to hide their
+privileges after a successful exploit.
+
+### Requirement
+
+Run Tetragon with `enable-process-creds` setting set to enable visibility
```
Comment (sharlns) [inline docs/content/en/docs/policy-library/observability/_index.md:173]: The same comment applies here. This level of configuration details doesn't belong to the "Policy Library".

**Review by sharlns — CHANGES_REQUESTED** [review]
LGTM, thank you! Please move the configuration details out from the "Policy Library" and just reference the corresponding page.

File: docs/content/en/docs/policy-library/observability/_index.md (hunk 1 above)
Comment (tixxdz) [reply docs/content/en/docs/policy-library/observability/_index.md:105]: > This level of configuration details doesn't belong to the "Policy Library" section. Can you please point just a reference here to the documentation where you can enable credentials in processX events?

I understand but this adds more work as right now we don't have this section, I will try to hack something up but don't want to delay more the PR.

Comment (tixxdz) [reply docs/content/en/docs/policy-library/observability/_index.md]: I added configuration section hope it's ok ;-)

**Comment by tixxdz** [comment]
> LGTM, thank you! Please move the configuration details out from the "Policy Library" and just reference the corresponding page.

Thanks @sharlns for the review, I did that. If it feels not perfect then let's discuss that in separate issue or PR please

**Review by sharlns — APPROVED** [review]
LGTM. Amazing work, thank you! 🙏

File: docs/content/en/docs/installation/configuration.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
+---
+title: "Configure Tetragon"
+linkTitle: "Configuration"
+weight: 5
+---
+
+## Configuration
```
Comment (mtardy) [inline docs/content/en/docs/installation/configuration.md:7]: you don't need to repeat the title
image

File: docs/content/en/docs/installation/configuration.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+{{< /tabpane >}}
+
+To read more about Tetragon configuration, please check our reference pages:
+
+* For Kubernetes deployments, see the [Helm chart]({{< ref "/docs/reference/helm-chart" >}}) reference.
+* For Container or systemd deployments, see the [Daemon configuration]({{< ref "/docs/reference/daemon-configuration" >}})
+reference.
+
+
+### Enable Process Credentials
```
Comment (mtardy) [inline docs/content/en/docs/installation/configuration.md:54]: if you remove the previous title, make those `##` (and the other one)

File: docs/content/en/docs/installation/configuration.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+{{< tabpane lang=shell >}}
+
+{{< tab Kubernetes >}}
+kubectl edit cm -n kube-system tetragon-config
+# Change your configuration setting, save and exit
+# Restart Tetragon daemonset
+kubectl rollout restart -n kube-system ds/tetragon
+{{< /tab >}}
+{{< tab docker >}}
```
Comment (mtardy) [inline docs/content/en/docs/installation/configuration.md:19]:
```suggestion
{{< tab Docker >}}
```

File: docs/content/en/docs/installation/configuration.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+{{< tabpane lang=shell >}}
+
+{{< tab Kubernetes >}}
+kubectl edit cm -n kube-system tetragon-config
+# Change "enable-process-cred" from "false" to "true", then save and exit
+# Restart Tetragon daemonset
+kubectl rollout restart -n kube-system ds/tetragon
+{{< /tab >}}
+{{< tab docker >}}
```
Comment (mtardy) [inline docs/content/en/docs/installation/configuration.md:68]:
```suggestion
{{< tab Docker >}}
```

**Review by mtardy — APPROVED** [review]
Thanks, I haven't checked the content just some nits on the form. 

If you could also wrap consistently the text around 80 chars that would be nice. Some paragraphs are in your PR while others aren't.

On another topic I find https://tetragon.io/docs/policy-library/observability/ really hard to read, maybe we could present that differently but that's another topic :)!

**Comment by tixxdz** [comment]
> Thanks, I haven't checked the content just some nits on the form.
> 
> If you could also wrap consistently the text around 80 chars that would be nice. Some paragraphs are in your PR while others aren't.

Thank you @mtardy I did and fixed all others ;-)
 
> On another topic I find https://tetragon.io/docs/policy-library/observability/ really hard to read, maybe we could present that differently but that's another topic :)!

Agree, need to not look at it for a moment then read it again ;-D !

**Pull request merged**